wrappers/obj-c/ODWLogManager.mm

```diff
@@ -161,7 +161,17 @@ +(nullable ILogger *)initializeLogManager:(nonnull NSString *)tenantToken withCo
 
         if (@available(iOS 10.0, macOS 10.12, *))
         {
-            strUserLocale = std::string([[locale languageCode] UTF8String]);
+            const char *localeCode = [[locale languageCode] UTF8String];
+
+            if (localeCode != nullptr)
+            {
+                strUserLocale = std::string(localeCode);
+            }
+            else
+            {
+                strUserLocale = "und";
+            }
+
```

Comment:
instead of using raw pointer, can we add size check if existing code as below:
```cpp
strUserLocale = std::string([[locale languageCode] UTF8String]);
if (strUserLocale.size() == 0) {
    strUserLocale = "und";
}
```